.github/workflows/haskell.yml

```diff
@@ -207,6 +212,7 @@ jobs:
             cabal_version: 3.6.2.0
             subdir: core
             ignore_error: false
+            rewrite_flags: dev use-unliftio debug
```

Comment:
should not remove `debug`, it can be used for debugging problems in production. Remove dev flags from both streamly and streamly-core. Can also remove `inspection`, `limit-build-mem`, and `opt`, `use-unfolds`, `use-folds`, `force-lstat-readdir`.